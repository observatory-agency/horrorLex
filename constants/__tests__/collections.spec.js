const collections = require('../collections');

describe('collections', () => {
  it('should match the snapshot', () => {
    expect(collections).toMatchSnapshot();
  });
});
