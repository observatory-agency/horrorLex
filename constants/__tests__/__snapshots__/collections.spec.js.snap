// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`collections should match the snapshot 1`] = `
Object {
  "books": Object {
    "index": Object {
      "author": "text",
      "isbn13": "text",
      "title": "text",
      "year": "text",
    },
    "name": "books",
  },
  "films": Object {
    "name": "films",
  },
}
`;
